server/controllers/work/editWork.js

```diff
@@ -0,0 +1,46 @@
+const { Work } = require('../../database/models');
+const { customError } = require('../errors');
+const { workEditValidation } = require('../../utils/validation');
+const { uploadImage, deleteImage } = require('../../utils');
+
+const editWork = async (req, res, next) => {
+  try {
+    const {
+      id, title, content,
+    } = await workEditValidation.validateAsync({ ...req.body, workImg: req.files.workImg });
+    const { providerID } = req;
+    const work = await Work.findByPk(id);
+    if (!work) {
+      throw customError('work does not exist', 400);
+    }
+    if (work.userId !== +providerID) {
```

Comment:
```suggestion
    const { userId, image, id } = work;
     if (work.userId !== +providerID) {
```
Work really updated after update, so we have to save the image value we want to get the key from, On this place